src/commands/synthetics/interfaces.ts

```diff
@@ -472,13 +473,10 @@ export interface Payload {
   options?: BatchOptions
 }
 
-export interface BaseTestPayload extends ServerConfigOverride {
-  executionRule?: ExecutionRule
```

Comment:
The `executionRule?: ExecutionRule` is already in `BaseConfigOverride`, which is extended by `ServerConfigOverride`.